### ISSUE TITLE
template name, for parse errors.

### ISSUE THREAD

**Issue by ryanong** [stefankroes/scribble#7]
as I am developing templates and doing integration I am getting a lot of parse errors the issue is however I have no idea where they are because each template lacks a name. It would make debugging a lot simpler if the errors had access to the template name.

**Comment by stefankroes** [comment]
I agree, this would be very useful! I'm willing to take a look at this but it might be a few weeks before I have time.
